testcases/two_zone_apartment_hydronic/models/config.json

```diff
@@ -0,0 +1,9 @@
+{"name"         : "two_zone_apartment_hydronic",
```

Comment:
Might be picky, but suggest ``twozone`` instead of ``two_zone`` to be consistent with other test case naming conventions.

Reply:
Changed name of testcase to `twozone`